allen-cell-animated/cell-feature-data
# Feature/publish to production

Problem
=======
What is the problem this work solves, including
closes #56 

Solution
========
What I/we did to solve this problem
- created a new manually triggered workflow that will run `release-data`, only on `main`
- Inputs include: `megaset-name`, `dataset-id` and `database`
- added a new github repo secret `owners`, so only authorized users are able to trigger this action  

with @meganrm 

## Type of change
Please delete options that are not relevant.
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Go to Actions  and choose `Release dataset to production DB` in workflows 
2. Click on `Run Workflow` and select `feature/publish-to-production` branch for testing. Note that the job is expected to be skipped on a non-main branch, comment out `if:  github.ref == 'refs/heads/main'` to run the job
3. Enter inputs and stay with default `staging` database for testing
4. Run workflow
